core-bundle/contao/templates/backend/be_login.html5

```diff
@@ -30,7 +30,7 @@
   <?php $this->block('container'); ?>
     <div id="container">
       <main id="main">
-        <form class="tl_login_form" method="post">
+        <form class="tl_login_form" method="post" data-controller="contao--webauthn" data-contao--webauthn-request-result-url-value="<?= $this->route('webauthn.controller.security.contao_backend.request.result') ?>" data-contao--webauthn-request-options-url-value="<?= $this->route('webauthn.controller.security.contao_backend.request.options') ?>" data-contao--webauthn-request-success-redirect-uri-value="<?= $this->webauthnSuccessUrl ?>">
```

Comment:
As discussed, we want to add the button below the existing login form. Then the username and password fields can remain required.